packages/shared/src/server/repositories/definitions.ts

```diff
@@ -9,8 +9,14 @@ export const clickhouseStringDateSchema = z
 
 //https://clickhouse.com/docs/en/integrations/javascript#integral-types-int64-int128-int256-uint64-uint128-uint256
 // clickhouse returns int64 as string
-export const UsageCostStringSchema = z.record(z.string(), z.string());
-export const UsageCostNumberSchema = z.record(z.string(), z.coerce.number());
+export const UsageCostStringSchema = z.record(
+  z.string(),
+  z.coerce.number().nullish(),
+);
+export const UsageCostNumberSchema = z.record(
+  z.string(),
+  z.coerce.number().nullish(),
```

Comment:
question: Why would we need to loosen the constraint here? I think there are great benefits to having all existing keys a number attached rather than needing to check for nullish values